bpf/lbmap/ipv6.go

```diff
@@ -58,12 +58,21 @@ func NewService6Key(ip net.IP, port uint16, slave uint16) *Service6Key {
 	return &key
 }
 
-func (k Service6Key) IsIPv6() bool           { return true }
-func (k Service6Key) Map() *bpf.Map          { return Service6Map }
-func (k Service6Key) NewValue() bpf.MapValue { return &Service6Value{} }
+func (k Service6Key) IsIPv6() bool               { return true }
+func (k Service6Key) Map() *bpf.Map              { return Service6Map }
+func (k Service6Key) NewValue() bpf.MapValue     { return &Service6Value{} }
+func (k *Service6Key) GetKeyPtr() unsafe.Pointer { return unsafe.Pointer(k) }
+func (k *Service6Key) GetPort() uint16           { return k.Port }
+func (k *Service6Key) SetPort(port uint16)       { k.Port = port }
+func (k *Service6Key) SetBackend(backend int)    { k.Slave = uint16(backend) }
+func (k *Service6Key) GetBackend() int           { return int(k.Slave) }
+
+func (k *Service6Key) Convert() {
+	k.Port = common.Swab16(k.Port)
+}
 
-func (k Service6Key) GetKeyPtr() unsafe.Pointer {
-	return unsafe.Pointer(&k)
+func (k *Service6Key) String() string {
```

Comment:
Does the String() interface works with a pointer to the receiver?
```go
svc:=Service6Key{}
fmt.Printf("%s", svc)
```
if this works as is then forget my comment